<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/menu_icon/icon/settings_icon.png</normaloff>:/menu_icon/icon/settings_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background: white; 
}
QTextEdit {
	border: 1px solid #c4cde0;
}
QPushButton {
	padding: 5px 10px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Segoe UI</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">padding: 2px 5px;</string>
         </property>
         <property name="text">
          <string>Max Connections </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_connection">
         <property name="font">
          <font>
           <family>Segoe UI</family>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>335</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>File Types</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="tab_doc">
          <attribute name="title">
           <string>Doucments</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>File Extensions ( seperated by comma)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="tb_document"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_compressed">
          <attribute name="title">
           <string>Compressed</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>File Extensions ( seperated by comma)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="tb_compressed"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_music">
          <attribute name="title">
           <string>Music</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>File Extensions ( seperated by comma)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="tb_music"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_video">
          <attribute name="title">
           <string>Video</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>File Extensions ( seperated by comma)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="tb_video"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_picture">
          <attribute name="title">
           <string>Picture</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>File Extensions ( seperated by comma)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="tb_picture"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_app">
          <attribute name="title">
           <string>Applications</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>File Extensions ( seperated by comma)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="tb_app"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Apply </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
